README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
+        out Employee[] output)
+    {
+        output = new Employee[2];
+        var employee = new Employee();
+        employee.EmployeeId = 1;
+        employee.FirstName = "Hello";
+        employee.LastName = "World";
+        employee.Company = "Microsoft";
+        employee.Team = "Functions";
+        output[0] = employee;
+        employee = new Employee();
+        employee.EmployeeId = 2;
+        employee.FirstName = "Hi";
+        employee.LastName = "SQLupdate";
+        employee.Company = "Microsoft";
+        employee.Team = "Functions";
+        output[1] = employee;
+
+        return new CreatedResult($"/api/addemployees-array", output);
+    }
+    ```
+
+    *In the above, "dbo.Employees" is the name of the database our output binding is upserting into. The line below is similar to the input binding and specifies where our SqlConnectionString is. For more information on this, see the [Output Binding Samples](#Output-Binding-Samples) section*
+
+- Hit 'F5' to run your code. Click the link to upsert the output array values in your SQL table. Your upserted values should launch in the browser.
+- Congratulations! You have successfully created your first SQL output binding! Checkout [Output Binding Samples](#Output-Binding-Samples) for more information on how to use it and explore on your own!
+
+### Trigger Tutorial ###
+
+This tutorial requires that the Azure SQL database is setup as shown in [Create Azure SQL Database](#Create-Azure-SQL-Database), and that you have the 'Employee.cs' class from the [Input Binding Tutorial](#Input-Binding-Tutorial).
+
+- Create a new file
+- Add the following namespaces
+
+    ```csharp
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using System.Collections.Generic;
+    using Microsoft.Azure.WebJobs.Extensions.Sql;
+    ```
+
+- Below, add the SQL trigger
+
+    ```csharp
+    namespace Company.Function
+    {
+        public static class EmployeeTrigger
+        {
+            [FunctionName("EmployeeTrigger")]
+            public static void Run(
+                [SqlTrigger("[dbo].[Employees]", ConnectionStringSetting = "SqlConnectionString")]
+                IEnumerable<SqlChangeTrackingEntry<Employee>> changes,
+                ILogger logger)
+            {
+                foreach (var change in changes)
+                {
+                    Employee employee = change.Data;
+                    logger.LogInformation($"Change occurred to Employee table row: {change.ChangeType}");
+                    logger.LogInformation($"EmployeeID: {employee.EmployeeId}, FirstName: {employee.FirstName}, LastName: {employee.LastName}, Company: {employee.Company}, Team: {employee.Team}");
+                }
+            }
+        }
+    }
     ```
 
-1. See samples below for information on how to use the binding.
+- Add the below to 'local.settings.json' in "Values"
+
+    ```json
+    "AzureWebJobsDashboard": "UseDevelopmentStorage=true"
```

Comment:
do they need this for all of them, or just the trigger?


We should

Reply:
I mainly ran into this when setting up the trigger. I'll add this to the quickstart setup though